frontend/packages/dev-console/src/components/topology/components/nodes/OperatorBackedServiceGroup.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import {
+  Node,
+  observer,
+  WithSelectionProps,
+  useDragNode,
+  Layer,
+  useHover,
+  createSvgIdUrl,
+  useCombineRefs,
+} from '@console/topology';
+import SvgBoxedText from '../../../svg/SvgBoxedText';
+import { nodeDragSourceSpec } from '../../componentUtils';
+import { TYPE_OPERATOR_BACKED_SERVICE } from '../../const';
+import useSearchFilter from '../../filters/useSearchFilter';
+import NodeShadows, { NODE_SHADOW_FILTER_ID, NODE_SHADOW_FILTER_ID_HOVER } from '../NodeShadows';
+
+export type OperatorBackedServiceGroupProps = {
+  element: Node;
+} & WithSelectionProps;
+
+const OperatorBackedServiceGroup: React.FC<OperatorBackedServiceGroupProps> = ({
+  element,
+  selected,
+  onSelect,
+}) => {
+  const [hover, hoverRef] = useHover();
+  const [innerHover, innerHoverRef] = useHover();
+  const [{ dragging, regrouping }, dragNodeRef] = useDragNode(
+    nodeDragSourceSpec(TYPE_OPERATOR_BACKED_SERVICE, false),
+    {
+      element,
+    },
+  );
+  const [{ dragging: labelDragging, regrouping: labelRegrouping }, dragLabelRef] = useDragNode(
+    nodeDragSourceSpec(TYPE_OPERATOR_BACKED_SERVICE, false),
+    {
+      element,
+    },
+  );
+
+  const nodeRefs = useCombineRefs(innerHoverRef, dragNodeRef);
+  const { data } = element.getData();
+  const [filtered] = useSearchFilter(element.getLabel());
+  const { x, y, width, height } = element.getBounds();
+
+  return (
+    <g
+      ref={hoverRef}
+      onClick={onSelect}
+      className={classNames('odc-operator-backed-service', {
+        'is-selected': selected,
```

Comment:
On selecting expanded group, it is not adding the highlight which is added when the group is collapsed.

Reply:
Fixed

Reply:
You actually don't need this line because it is only needed on the `__bg` node.